lib/v2/lightnovel/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/:keywords/:security_key': ['nightmare-mio'],
```

Comment:
https://github.com/DIYgod/RSSHub/blob/86c458f9d80d92495e46c04ff7a3650f53390264/lib/v2/lightnovel/router.js#L2
```suggestion
    '/:keywords/:security_key?': ['nightmare-mio'],
```